src/validator/CreditCardVendorsReader.java

```diff
@@ -0,0 +1,58 @@
+package validator;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class CreditCardVendorsReader implements CreditCardVendorReading {
+	@Override
+	public ArrayList<CreditCardVendor> returnVendorArray() {
+		ArrayList<CreditCardVendor> list = new ArrayList<CreditCardVendor>();
```

Comment:
From Java 7 the diamond operator (<>) does not need repetition of the used type. You can just write: 
`new ArrayList<>();`

The same on lines 19 and 20.

Reply:
It's better to call variables by names that business-wise signify what it is. In this case, it should be creditCardVendors rather than list.